plugins/inputs/ceph/ceph.go

```diff
@@ -531,17 +610,24 @@ func decodeDf(acc telegraf.Accumulator, input string) error {
 type CephOSDPoolStats []struct {
 	PoolName     string `json:"pool_name"`
 	ClientIORate struct {
-		ReadBytesSec  float64  `json:"read_bytes_sec"`
-		WriteBytesSec float64  `json:"write_bytes_sec"`
-		OpPerSec      *float64 `json:"op_per_sec"` // This field is no longer reported in ceph 10 and later
```

Comment:
Please keep old fields for backward compatibility.

Reply:
Re-added with latest changes.